src/landpatterns/BGA/package.stanza

```diff
@@ -106,6 +176,45 @@ public defn BGA (
     density-level
   )
 
+doc: \<DOC>
+@title BGA Package Constructor
+@desc This function constructs a BGA package with the passed arguments.
+@param num-leads Total number of leads for the BGA Grid
+@param lead-diam BGA Ball Lead Diameter in mm
+@param package-body Component body physical representation.
+@param pad-planner Pad Planner for the BGA package
+@param lead-numbering Lead Numbering Scheme for the BGA Package
+@param density-level Density Level for the Generated Package
```

Comment:
This documentation does not provide a lot of value. These statements are tautological at best.